libraries/AP_ESC_Telem/AP_ESC_Telem.cpp

```diff
@@ -107,7 +107,7 @@ uint32_t AP_ESC_Telem::get_active_esc_mask() const {
             // have never seen telem from this ESC
             continue;
         }
-        if (now - _telem_data[i].last_update_ms >= ESC_TELEM_DATA_TIMEOUT_MS
+        if ((_telem_data[i].last_update_ms == 0 || now - _telem_data[i].last_update_ms >= ESC_TELEM_DATA_TIMEOUT_MS)
```

Comment:
I though @WickedShell already had a helper for this?

Reply:
I do think a helper woulud make this clearer

Reply:
added stale() helper